libs/vkd3d-shader/checksum.c

```diff
@@ -1,303 +1,118 @@
 /*
- * Copyright (C) 2001 Nikos Mavroyanopoulos
- * Copyright (C) 2004 Hans Leidekker
+ * Copyright (c) 2022 Hans-Kristian Arntzen for Valve Corporation
  *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
+ * SPDX-License-Identifier: MIT
```

Comment:
If we are adding a SPDX-License-Identifier... Can we just drop our whole license text past where that is.

Kernel, some new parts of Mesa and Khronos stuff does that.

(obv cant do it from the part taken from RDoc)

Reply:
Fixed.